addons/website/views/res_config_settings_views.xml

```diff
@@ -370,7 +370,10 @@
                                     <div class="text-muted mt8">
                                         Sitemap.xml: Help search engine crawlers to find out what pages are present and which have recently changed, and to crawl your site accordingly. This file is automatically generated by Odoo.
                                     </div>
-                                    <div class="mt4">
+                                    <div class="alert alert-primary mt4" role="alert">
+                                        Google doesn't need to be pinged anymore. It will automatically fetch your /sitemap.xml.
+                                    </div>
+                                    <div class="mt4 d-none">
```

Comment:
I'll leave that one up to you, I would keep it shown otherwise for new db it seems weirder than if we kept it shown imo:
![image](https://github.com/odoo/odoo/assets/30048408/be44136d-1cd2-4864-b724-89786bde8616)
vs
![image](https://github.com/odoo/odoo/assets/30048408/1bd92235-7323-4c3e-8557-f4d49cc83f69)

But your call, either resolve the comment and don't change it, or change it I don't mind